CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.1.4] - 2021-05-21
+
+- Add LICENSE
+
 ## [1.1.3] - 2021-05-20
 
 - Change icons names and selects storys data
```

Comment:
https://github.com/codespell-project/codespell tells me that `storys` is not a word.

Pick one:
* [ ] `stories` -- Plural but not possessive
* [ ] `story's` -- Singular possessive
* [ ] `stories'` -- Plural possessive

Reply:
Neither of them. I think the whole sentence was a bit hard to understand so I changed it. Thanks for pointing it out!